openlibrary/core/vendors.py

```diff
@@ -276,10 +277,28 @@ def get_betterworldbooks_metadata(isbn):
     isbn = normalize_isbn(isbn)
     try:
         if isbn:
-            return _get_betterworldbooks_metadata(isbn)
+            metadata = _get_betterworldbooks_metadata(isbn)
+            if not metadata.get('price') and thirdparty:
+                return _get_betterworldbooks_thirdparty_metadata(isbn)
+            return metadata
     except Exception:
         return {}
 
+def _get_betterworldbooks_thirdparty_metadata(isbn):
```

Comment:
Let's create a test for this method, that fails if BWB changes their HTML. It'll mean making a network request in our tests (which is usually bad practice), but I would rather have us know if this stops working.